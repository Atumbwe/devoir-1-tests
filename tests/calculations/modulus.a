534 76 mod