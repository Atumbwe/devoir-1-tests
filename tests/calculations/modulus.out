 ___
< 2 >
 ---
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
